### PR TITLE
Fixed token parse error

### PR DESCRIPTION
**Pull request by Sniedes722**
Token handler was returning a 'list of out range' Index Error, so I added token as a Request attribute and returned it in a manner identical to the ip handler.

### CODE REVIEW

**Comment by seemethere** [comment]
I restarted the build, let's see if that fixes it.

**Comment by Sniedes722** [comment]
@seemethere probably related to [this](https://www.traviscistatus.com/incidents/hx7cnxbch9xf).

**Comment by seemethere** [comment]
Well that sucks 😢

File: sanic/request.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ class Request(dict):
         'app', 'url', 'headers', 'version', 'method', '_cookies', 'transport',
         'query_string', 'body',
         'parsed_json', 'parsed_args', 'parsed_form', 'parsed_files',
-        '_ip',
+        '_ip','_token',
```
Comment (r0fls) [inline sanic/request.py]: there should be a space after the comma

Comment (r0fls) [inline sanic/request.py]: i.e. `'_ip', '_token',`

File: sanic/request.py [hunk 2]
```diff
@@ -79,10 +79,9 @@ def token(self):
 
         :return: token related to request
         """
-        auth_header = self.headers.get('Authorization')
-        if auth_header is not None:
-            return auth_header.split()[1]
-        return auth_header
+        if not hasattr(self, '_token'):
+            self._token = self.headers.get('Authorization')
+        return self._token
```
Comment (r0fls) [inline sanic/request.py]: This logic can be condensed into one line without sacrificing anything:

```
self._token = getattr(self, '_token', self.headers.get('Authorization'))
```

**Comment by seemethere** [comment]
@Sniedes722 looks like the logs on TravisCI are back up. You have one failing unittest and a flake8 error

**Comment by Sniedes722** [comment]
@seemethere thanks. I'll fix tonight and merge upstream.

**Comment by Sniedes722** [comment]
Closed in favor of #649

**Pull request closed without merging**